docs/index.md

```diff
@@ -5,52 +5,42 @@ page_keywords: documentation, docs,  docker, compose, orchestration, containers
 
 ## Overview
 
-Compose is a tool that allows you to orchestrate multiple Docker containers.
-With Compose, you can build clusters of containers which provide the resources
-(services, volumes, etc.) needed to build and run a complete distributed
-application.
+Compose is a tool for defining and running complex applications with Docker.
+With Compose, you define a multi-container application in a single file, then
+spin your application up in a single command which does everything that needs to
+be done to get it running.
 
-You can use Compose to build your app with containers hosted locally, or on a
-remote server, including cloud-based instances - anywhere a Docker daemon can
-run. Its primary use case is for development environments, but it can be used
-just as easily for staging or CI.
+Compose is especially good for development environments, staging servers and CI.
+It can be used in production, but it isn't (yet) recommended that you do so.
+We're actively working on making it better in that regard - see the
+[roadmap](https://github.com/docker/fig/blob/master/ROADMAP.md).
 
-Using Compose is basically a three-step process.
-
-First, you define your app's environment with a `Dockerfile` so it can be
-reproduced anywhere:
-
-    FROM python:2.7
-    WORKDIR /code
-    ADD requirements.txt /code/
-    RUN pip install -r requirements.txt
-    ADD . /code
-
-Next, you define the services that make up your app in `docker-compose.yml` so
-they can be run together in an isolated environment:
+Using Compose is a two-step process. First, define the services that make up
+your app in `docker-compose.yml`, using standard Docker configuration options:
```

Comment:
I feel its value has diminished since we put it in (back when we were still explaining what Docker was).

Let's be explicit: how much Docker knowledge do we expect people landing on this page to have?